src/tribler/core/check_os.py

```diff
@@ -66,22 +66,28 @@ def set_process_priority(pid=None, priority_order=1):
     """
     if priority_order < 0 or priority_order > 5:
         return
-
-    process = psutil.Process(pid if pid else os.getpid())
+    if sys.platform not in {'win32', 'darwin', 'linux2', 'linux'}:
```

Comment:
**Nitpick:** It looks like the "linux2" value is only possible to get from `sys.platform` on Linux machines when using Python <= 3.3: https://stackoverflow.com/questions/446209/possible-values-from-sys-platform

As we use Python 3.8, we can only get "linux" value from `sys.platform` on Linux machines. So we can remove `linux2` from the list.

That also means that the previous code, if I understand correctly, never calls `process.nice` on Linux after upgrading Tribler from Python 3.3 to a more recent Python version.

Reply:
Removed.

> That also means that the previous code, if I understand correctly, never calls process.nice on Linux after upgrading Tribler from Python 3.3 to a more recent Python version.

Indeed, I share your understanding.